### DISCUSSION TITLE
Add separate framework for actix-web 4.0

### DISCUSSION THREAD

**Discussion by patrickfreed** [TechEmpower/FrameworkBenchmarks#7013] [Q&A]
I'd like to submit a benchmark for the [MongoDB Rust driver](https://github.com/mongodb/mongo-rust-driver) used along side [actix-web](https://github.com/actix/actix-web), but the driver currently only works with actix-web 4.0, and the [existing actix framework tests](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Rust/actix/Cargo.toml#L28) use an earlier version. I'd rather not go through all the effort of updating all the existing actix tests, so would it be okay if I introduced actix-4 as a new framework to test against for the MongoDB driver? Other contributors could then port the other tests to it too.

**Comment by robjtede** [comment]
the Actix Web tests in-tree now use v4 RC
